Sources/MapboxMaps/Style/Style.swift

```diff
@@ -430,4 +435,18 @@ public struct StyleTransition: Codable {
         self.duration = duration
         self.delay = delay
     }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        duration = try container.decode(Double.self, forKey: .duration) / 1000
+        delay = try container.decode(Double.self, forKey: .delay) / 1000
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+
+        duration = try container.encode(Double.self, forKey: .duration) * 1000
+        delay = try container.encode(Double.self, forKey: .delay) * 1000
```

Comment:
```suggestion
        try container.encode(duration, forKey: .duration) * 1000
        try container.encode(delay, forKey: .delay) * 1000
```

Reply:
This change does not work, it brings a compiler error and we need to encode the key so I don't see the purpose of this change